cli/kata-check_ppc64le.go

```diff
@@ -75,3 +94,23 @@ func archKernelParamHandler(onVMM bool, fields logrus.Fields, msg string) bool {
 func getCPUDetails() (vendor, model string, err error) {
 	return genericGetCPUDetails()
 }
+
+func isSMTOff() bool {
+
+	// Check if the SMT is available and off
+
+	cmd := exec.Command(ppc64CpuCmd, smtStatusOption)
+	additionalEnv := "LANG=C"
+	newEnv := append(os.Environ(), additionalEnv)
```

Comment:
Nit: This is changing the environment in the parent but you actually only need to change the child (`cmd.Env`). This is effectively equivalent today but if we were to i18n the runtime, this would become a bug :)

Reply:
@jodh-intel : I am sending an updated PR.

Reply:
@jodh-intel : Updated my PR.

Reply:
Thanks @nitkon.

lgtm